src/Microsoft.Azure.WebJobs/IBinder.cs

```diff
@@ -1,26 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Microsoft.Azure.WebJobs
-{
-    /// <summary>
-    /// Interface defining functionality for dynamically binding to WebJobs SDK attributes
-    /// at runtime.
-    /// </summary>
-    /// <remarks>This interface enables imperative binding with attribute information specified at runtime.</remarks>
-    public interface IBinder
```

Comment:
Removing this interface would be a significant breaking change to all consumers (and not just extension authors). This has been a long-standing interface that we've encouraged consumers to use to do late bindings.